src/client/GameWindow.cpp

```diff
@@ -0,0 +1,41 @@
+#include <client.hpp>
+#include <iostream>
+
+#define MAP_X_OFFSET 0
+#define MAP_Y_OFFSET 0
+
+namespace client {
+
+GameWindow::GameWindow() {
+    window.create(sf::VideoMode(1245, 725),"Civilization VII");
+    map.setOffset(MAP_X_OFFSET, MAP_Y_OFFSET);
```

Comment:
we should change the name of MapDisplay instance in GameWindow (in class-diagram.dia) because "map" is the name of the class "Map".